shells/dev/src/devtools.js

```diff
@@ -78,7 +78,10 @@ inject('./build/app.js', () => {
 
         // Initialize the backend only once the DevTools frontend Store has been initialized.
         // Otherwise the Store may miss important initial tree op codes.
-        inject('./build/backend.js');
+        inject('./backend.js', () => {
+          // Clear noisy webpack "WDS: ..." output.
+          console.clear();
```

Comment:
This is gross but webpack is hopeless. I've spent an hour trying to get it to shut up with these logs, and I didn't reach success. It seems like some race condition is preventing the silent config from kicking in, so I just clear all initial logs. Could be annoying if you debug the init path but I figured it's not too bad and it's always clearly shown in DevTools that console was cleared — so you can always comment it out.